jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/DelayedHandler.java

```diff
@@ -146,7 +147,7 @@ public void accept(Fields fields, Throwable throwable)
         }
     }
 
-    public static class UntilMultiPart extends DelayedHandler
+    public static class UntilFormData extends DelayedHandler
```

Comment:
Is this name correct? it implies it can handle normal non-multipart form data.  Can it?